southworkscom/AI
# [Enterprise Bot Template][TS] Update Dialogs with Resources

## Description

- Update **Dialogs**.
- Rename **utils** to **extensions** folder.
- Replace **signin** for **authentication** folder.
- Update **<name_of_dialog>Strings** resources.
- Add **Intro** resources.
- Rename **interruptableStatus** to **interruptionStatus** file.

![image](https://user-images.githubusercontent.com/11904023/51185027-cf250300-18b4-11e9-98ed-077a933e2de0.png)

## Related Issue
Fixes 536.

## Testing Steps

1. Open a new terminal in `AI/templates/Enterprise-Template/src/typescript`.
2. Run `msbuild` using the instruction in [DEPLOYMENT.md](https://github.com/Microsoft/AI/blob/master/templates/Enterprise-Template/src/typescript/deploymentScripts/DEPLOYMENT.md). 
3. Run `npm install` to install dependencies.
4. Run `npm run build` to build the solution.
5. Run `npm run start` to start the bot.
6. Talk to the bot and check the behavior.

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
